### ISSUE TITLE
Genome Browser Preview not rendering on gene, variation, PCR product pages

### ISSUE THREAD

**Issue by chris-grove** [WormBase/website#7286]
The "Genome Browser Preview" image is not rendering in the "Location" widget on gene pages, variation pages, or PCR product pages.

<img width="808" alt="Screenshot 2019-09-10 10 53 45" src="https://user-images.githubusercontent.com/611382/64625082-985aaa80-d3b9-11e9-8349-68c1527ca52e.png">
<img width="753" alt="Screenshot 2019-09-10 10 53 38" src="https://user-images.githubusercontent.com/611382/64625083-985aaa80-d3b9-11e9-8d55-50ca252f9b50.png">

**Comment by sibyl229** [comment]
Hi @scottcain the URL that the widget uses http://www.wormbase.org/tools/genome/jbrowse-simple/full.html?tracks=Curated_Genes%2CClassical_alleles%2CYACs_Fosmids_Cosmids&data=data%2Fc_elegans_simple&nav=0&overview=0&loc=III%3A5688790..5693529&tracklist=0 

It seems to be getting a 404 error through by the apache server. Thoughts?

**Comment by scottcain** [comment]
super weird--a symlink got deleted somehow; I restored it, but now images are getting generated but they're empty.  Hopefully will have a fix shortly.

**Comment by scottcain** [comment]
OK, something I did when I modified the build script for 272 cause a file to not be created that only affected the embedded jbrowse images.  Fortunately, that file doesn't really change from release to release, so I just got the one from 271.  So, this bug is technically resolved since it's working in production, but I'll leave this open in hopes of reminding me to fix the build script (I'm sure staging is currently broken in the same way right now).
